packages/vite/src/rsc/rscBuildAnalyze.ts

```diff
@@ -56,6 +53,7 @@ export async function rscBuildAnalyze(viteConfigPath: string) {
       rollupOptions: {
         onwarn: onWarn,
         input: {
+          // @TODO(RSC_DC): We could generate this entries file from the analyzedRoutes
```

Comment:
For sure, but having gone through this with @Josh-Walker-GM - I think for the dev server he will need to generate these entries on the fly - so we will need something like a plugin or virtual module to generate these entries, rather than write to actual file.